assets/scss/components/_button.scss

```diff
@@ -30,6 +30,14 @@ _(The `extend` directive can cause ugly cascade problems)_
 @include buttonBase();
 ```
 */
+
+//
+// Get these vars updated in sasstools before merging
```

Comment:
You can add the `Prelim` label and leave a note in the PR to help make sure this isn't merged prematurely